phase1-coordinator/src/environment.rs

```diff
@@ -225,21 +224,17 @@ pub struct Environment {
     /// Returns the maximum duration a contributor can go without
     /// being seen by the coordinator before it will be dropped from
     /// the ceremony by the coordinator.
-    #[serde_as(as = "DurationSecondsWithFrac<String>")]
-    contributor_seen_timeout: chrono::Duration,
+    contributor_seen_timeout: time::Duration,
     /// The maximum duration a verifier can go without being seen by
     /// the coordinator before it will be dropped from the ceremony by
     /// the coordinator.
-    #[serde_as(as = "DurationSecondsWithFrac<String>")]
```

Comment:
This broke some of the tests when I tried it, not sure if we should be serialising using the `serde-human-readable` feature flag on the `time` crate?